src/Dependencies/PooledObjects/ArrayBuilder.cs

```diff
@@ -634,6 +634,13 @@ public void AddRange(T[] items, int length)
             _builder.AddRange(items, length);
         }
 
+#if COMPILERCORE
+        public void AddRange(OneOrMany<T> items)
```

Comment:
>AddRange [](http://example.com/codeflow?start=20&length=8)

Is `RemoveLowerPriorityMembers()` the only caller of this method? If so, consider removing this method and calling `resultsForType.AddRangeTo(results);` directly in that caller.

Reply:
Yes, but I intentionally added this because the calling order is awkward and non-discoverable otherwise. We need AddRangeTo to make sure we avoid allocation, but no one is going to call it in that direction unless they already know it exists. I'd prefer to keep this method.